docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -15,13 +15,36 @@ in the [Working with command-line compiler](command-line.md) tutorial.
 
 Kotlin compilers have a number of options for tailoring the compiling process.
 
-Using a build script, you can specify additional compilation options. Use the `kotlinOptions` property of a Kotlin compilation task for it. 
+Using a build script, you can specify additional compilation options. Use the `compilerOptions` property of a Kotlin compilation task for it. 
 For example:
 
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
 ```kotlin
-compileKotlin.kotlinOptions.freeCompilerArgs += "-Xexport-kdoc"
+compileKotlin.compilerOptions.freeCompilerArgs.add("-Xexport-kdoc")
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+compileKotlin {
```

Comment:
@Tapchicoma I added a forgotten code for Groovy.

Reply:
looks ok